proto/skaffold.proto

```diff
@@ -222,7 +222,9 @@ message TriggerRequest {
 
 // TriggerState represents trigger state for a given phase.
 message TriggerState {
-  bool enabled = 1; // enable or disable a trigger state
+  oneof val {
+    bool enabled = 1; // enable or disable a trigger state
```

Comment:
lame that you can't just say `oneof bool enable = 1;` but 🤷

Reply:
🤷🏾‍♂️